test/visibility/import/submoduleCloser.bad

```diff
@@ -1,2 +1,6 @@
 submoduleCloser.chpl:11: In function 'main':
+submoduleCloser.chpl:14: warning: potentially surprising shadowing for 'Details'
+submoduleCloser.chpl:13: note: it refers to a symbol found through the 'use' of 'ExternalLib1' here
+submoduleCloser.chpl:2: note: leading to 'Details' defined here
+submoduleCloser.chpl:6: note: but, there is a shadowed symbol 'Details' defined here
```

Comment:
Since you're touching the test anyways, this is the one listed in #19782, so it'd make sense to resolve the future and have this PR close that issue